website/pages/docs/nia/api.mdx

```diff
@@ -0,0 +1,223 @@
+
+Below are all possible values for health status. See individual endpoints below for details on how each health status value is determined:
+ - Healthy
+ - Degraded
+ - Critical
+ - Undetermined
+
+### Overall Status
+
+This endpoint returns the overall status information across all tasks.
+
+Overall health status value is determined by the [health status values of all individual tasks](/docs/nia/api#task-status)
+ - Healthy: All tasks have health status "healthy".
+ - Degraded: At least one task has health status "degraded" but none are "critical".
+ - Critical: At least one task has health status "critical".
+ - Undetermined: No tasks have health status information at the moment.
+
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/status` | `application/json` |
+
+#### Request Parameters
+Currently no request parameters are offered for the overall status API.
+
+#### Response Fields
+
+* `status` - `(string)` Values are "healthy", "degraded", "critical", or "undetermined". This is determined by the health status values of all individual tasks, as described above.
+
+
+#### Example
+
+Request:
+```shell-session
+curl localhost:8501/v1/status
+```
+
+Response:
+```json
+{
+  "status": "healthy"
+}
+```
+
+### Task Status
+
+This endpoint returns the individual task status information for a single specified task or for all tasks.
+
+Task health status value is determined by the success or failure of all stored [event data](/docs/nia/tasks#event) on the process of updating network infrastructure for a task. Currently only the 5 most recent events are stored per task.
+ - Healthy: All stored events of the task are successful.
+ - Degraded: More than half of the stored events are successful _or_ less than half of the stored events are successful but the most recent event is successful.
+ - Critical: Half or less of the stored events are successful and the most recent event is not successful.
+ - Undetermined: No event data is stored for the task.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/status/tasks/:task` | `application/json` |
+
+#### Request Parameters
+
+* `task` - `(string)` Option to specify the name of the task to return in the response. If not specified, all tasks are returned in the response.
+* `include` - `(string)` Only accepts the value "events". Use to request including stored event information with response.
```

Comment:
> Use to request including stored event information with response.

This is a bit difficult to read, do you mind rewording it?

Reply:
:nod: what do you think about "Use to include stored event information in response."?

Reply:
👍